### DISCUSSION TITLE
mp3 with 192kbs

### DISCUSSION THREAD

**Discussion by TCB1974** [bbc/peaks.js#516] [General]
Hi all,

pretty sure this is just me not setting the options correctly. I have mp3s with 192kbs bitrate and with 128kbs bitrate. The ones with 128kbs are played without any issue, showing waveform, I can scroll forward / backward, etc. Trying to do the same with a 192kbs mp3 I do see the waveform, I can click around, but no audio. Pressing play button instantly stops and no error message. If I drag & drop the 192kbs mp3 directly into browser audio is played. What am I doing wrong? Thank you very much.

**Comment by chrisn** [comment]
Peaks.js doesn't do anything special regarding audio playback, it just responds to events from the HTML `<audio>` element.

**Comment by TCB1974** [comment]
thank you, Chris. This brought me to the correct place. Issue has been the proxy server that didn't pass the audio content correctly.

**Comment by chrisn** [comment]
I'm happy to hear you've resolved the problem. Thanks for letting us know.
